chain/chain/src/tests/simple_chain.rs

```diff
@@ -247,3 +248,41 @@ fn next_blocks() {
     assert_eq!(chain.mut_chain_store().get_next_block_hash(&b1_hash).unwrap(), b3_hash);
     assert_eq!(chain.mut_chain_store().get_next_block_hash(&b3_hash).unwrap(), b4_hash);
 }
+
+#[test]
```

Comment:
@wacban is this a good place to test?

Reply:
I would first try directly in block.rs but here is fine too.